macosx/PrefsController.h

```diff
@@ -78,7 +78,7 @@
 - (void) updatePortStatus;
 - (void) portCheckerDidFinishProbing: (PortChecker *) portChecker;
 
-- (NSArray *) sounds;
+@property (nonatomic, readonly) NSArray *sounds;
```

Comment:
As a followup, these should 1. specify the contained type, and 2. should be a `copy` property

Reply:
1. I will handle lightweight generics & ivars later in a separate PR, since they are not part of the 'Modern Objective-C' document I mentioned in the beginning, and the current PR is large enough already. Thank you for bringing this up!
2. @mikedld (the original reviewer) signalled multiple times above that `readonly` property should not be a `copy` property. I agree with this – why would a property without a setter require a specific setter behaviour (apart from the case where the property is privately `readwrite`)?

Reply:
Re: 2., I believe it makes sense to make any object that's potentially mutable and representing a value type as copy - in this case, it would signal that storing it means it won't change under the hood. But to be fair this is a dark corner of Objective-C that shows its weakness vs modern languages like Swift where this wouldn't be a concern, and in practice it doesn't matter.